packages/MSBot/docs/add-services.md

```diff
@@ -0,0 +1,284 @@
+## Connecting Your Bot to a Service
+
+To connect your bot to a service you run 
+
+```shell
+msbot connect [command]
+```
+
+Where the command is one of the services
+
+| Command                                                 | Description                              |
+|---------------------------------------------------------|------------------------------------------|
+| [appinsights](#connecting-to-azure-appinsights-service) | connect to Azure AppInsights             |
```

Comment:
This list list is the same as I get when I use the help command. I'm not sure if this is adding any value.

Reply:
I believe the list is useful for someone looking through the docs even before they've installed the tool.